src/main/Python/compute_var.py

```diff
@@ -0,0 +1,22 @@
+# D(X) = E[(X-EX)^2] = E(X^2) - (EX)^2
+# Here the formulas are used for the overall variance
+
+from typing import List 
```

Comment:
Format Line trailing space
```suggestion
from typing import List
```